sp800-63-3/sec1_2_introduction.md

```diff
@@ -48,8 +48,8 @@ their authentication assurance requirements:
 
 1.  *Conduct a risk assessment of the government system* - No specific
     risk assessment methodology is prescribed for this purpose;
-    however, NIST SP 800-30 [[SP 800-30]](#SP800-30) offers a general
-    process for risk assessment and risk mitigation, and NIST SP 800-37 Revision 1 [[SP 800-37]](#SP800-37) provides recommendations on the selection and specification of security controls for a system as part of an organization-wide information security program. This guideline supports the identification of risk to the organization or to individuals associated with the operation of a system.
+    however, NIST [[SP 800-30]](https://pages.nist.gov/800-63-3/sp800-63-3.html#SP800-30) offers a general
```

Comment:
References in -3 should be put in the -3 references section rather than linking to another volume.